ci/do-audit.sh

```diff
@@ -33,6 +33,9 @@ cargo_audit_ignores=(
 
   # curve25519-dalek
   --ignore RUSTSEC-2024-0344
+
+  # tonic
```

Comment:
Might be nice to add more to the comment. Something like:
"When using tonic::transport::Server there is a remote DoS attack that can cause the server to exit cleanly on accepting a tcp/tls stream.

Ignoring because we do not use this functionality."

Reply:
more specifically, we don't use the server side of `tonic` at all